sound/soc/sof/ipc4-loader.c

```diff
@@ -423,6 +423,62 @@ int sof_ipc4_reload_fw_libraries(struct snd_sof_dev *sdev)
 	return ret;
 }
 
+/**
+ * sof_ipc4_update_basecfg_from_manifest - Update the base config from manifest
+ * @sdev: SOF device
+ * @fw_module: pointer struct sof_ipc4_fw_module to parse
+ * @basecfg: Pointer to the base_config to update
+ *
+ * Note: currently only used to update the CPC value if it is not set from
+ * topology already.
```

Comment:
but why would we need to update the base config? You're not describing why we would need to modify a base config?

Reply:
New version is limited to basecfg->cpc updating, so the Note is dropped.